docs/programming/MPI-C.md

```diff
@@ -8,10 +8,8 @@ Intel C++ Compiler, GCC, IntelMPI, and OpenMPI to create a
 multiprocessor ‘hello world’ program in C++.  This tutorial assumes
 the user has experience in both the Linux terminal and C++.
 
-__Resources:__
-- [http://www.dartmouth.edu/~rc/classes/intro_mpi/intro_mpi_overview.html](http://www.dartmouth.edu/~rc/classes/intro_mpi/intro_mpi_overview.html)  
+__Resource:__
 - [http://mpitutorial.com/tutorials/](http://mpitutorial.com/tutorials/)  
-- [http://condor.cc.ku.edu/~grobe/docs/intro-MPI-C.shtml](http://condor.cc.ku.edu/~grobe/docs/intro-MPI-C.shtml)
 
 
 ## Setup and “Hello, World”
```

Comment:
This may have been resolved already, but could you please make sure that issue #272 has been addressed. It might have been as it is old.